lib/ACF/DynamicEventGroup.php

```diff
@@ -286,10 +286,10 @@ protected function get_choices_by_name( string $name, array $field ) : array {
 
         $cache_key = 'events-' . $name;
         $response  = wp_cache_get( $cache_key );
+        $lang_key  = Localization::get_current_language();
 
-        if ( ! $response ) {
+        if ( ! $response && ( $lang_key === 'fi' || $lang_key === 'en' ) ) {
```

Comment:
Nämä on yleensä parempi tehdä in_array():lla, niin on laajennettavissa myös kivasti jälkeenpäin.


```suggestion
        $request_allowed_langs = [
            'fi',
            'en',
        ];

        if (
            ! $response &&
            in_array( $lang_key, $request_allowed_langs, true )
        ) {
```